lib/compress/zstd_ldm.c

```diff
@@ -620,7 +620,7 @@ size_t ZSTD_ldm_blockCompress(rawSeqStore_t* rawSeqStore,
 
     assert(rawSeqStore->pos <= rawSeqStore->size);
     assert(rawSeqStore->size <= rawSeqStore->capacity);
-    /* Loop through each sequence and apply the block compressor to the lits */
+    /* Loop through each sequence and apply the block compressor to the lists */
```

Comment:
This isn't a typo: `lits` is correct

Reply:
It could be extended from `lits` to `literals`

Reply:
fixed, thanks for the hint.